### ISSUE TITLE
Check dependencies for updates on rancher/dashboard

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#4534]
With Nuxt removed, let's go through our other dependencies in order of importance and check for newer compatible versions and upgrade.

If necessary, create separate issues for specific packages if the work involved is non-trivial.

**Comment by nwmac** [comment]
Blocked on nuxt removal

**Comment by nwmac** [comment]
Probably good to do after nuxt removal, since we will remove some dependencies then

**Comment by gaktive** [comment]
Bump nodejs should be the first thing. We'll need to update the Drone config to ensure we include the latest when we ship.

**Comment by torchiaf** [comment]
@nwmac we are still using `nuxt`. Can we go ahead anyway ?

**Comment by cnotv** [comment]
For the records: @nwmac the libraries from the report in the Jira ticket talk about `Handlebars.js` and `Bootstrap `.

**Comment by nwmac** [comment]
Yeah I know,  they are updated on the api-ui repo. But generally we want to update dependencies to newer versions where possible.

**Comment by cnotv** [comment]
There's not an higher version for handlebar.js and is not present in Rancher anyhow.
We have already an issue to remove jQuery. 
Bootstrap is not present in out application.

Should we consider this then close and just write in Jira?

**Comment by gaktive** [comment]
Will move https://github.com/rancher/dashboard/pull/8715 into a new ticket. Everything else here is taken care of.
